src/core/memory.cc

```diff
@@ -1548,25 +1548,25 @@ void configure(std::vector<resource::memory> m, bool mbind,
     }
 }
 
-statistics stats() {
+statistics stats() noexcept {
```

Comment:
This throws when seastar is compiled with default allocator.

Reply:
where?